src/org/sosy_lab/java_smt/api/FormulaType.java

```diff
@@ -478,7 +594,8 @@ public static FormulaType<?> fromString(String t) {
     } else if (t.startsWith("FloatingPoint<")) {
       // FloatingPoint<exp=11,mant=52>
       List<String> exman = Splitter.on(',').limit(2).splitToList(t.substring(14, t.length() - 1));
-      return FormulaType.getFloatingPointType(
+      // SMTLIB2 standard expects the hidden bit to be part of the mantissa
+      return FormulaType.getFloatingPointTypeFromSizesWithHiddenBit(
```

Comment:
Why is there a reference to SMTLIB2 here? This method has nothing to do with the SMTLIB standard.

What `toString()` and `fromString()` do is defined only by JavaSMT.